File: miraculous-backend/src/main/java/com/example/miraculousbackend/dto/CommentDispatchDto.java
package com.example.miraculousbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CommentDispatchDto {

    private long authorId;
    private String text;
    private long artistId;
    private long albumId;
    private long trackId;

}

File: miraculous-frontend/miraculous-ui/src/dto/comment-dispatch-dto/comment-dispatch-dto.ts
export class CommentDispatchDto {

  authorId: number;
  text: string;
  artistId?: number;
  trackId?: number;
  albumId?: number;


  constructor(authorId?: number, text?: string, artistId?: number, trackId?: number, albumId?: number) {
    this.authorId = authorId;
    this.text = text;
    this.artistId = artistId;
    this.trackId = trackId;
    this.albumId = albumId;
  }
}
